### DISCUSSION TITLE
Karpenter & Crossplane

### DISCUSSION THREAD

**Discussion by prophesarns** [crossplane/crossplane#3038] [Q&A]
Hi,

I tried to find some information regarding using crossplane and karpenter on AWS. 
Will crossplane and Karpenter conflict ? Karpenter will up/down scale nodes from a K8S cluster and i wonder if crossplane will try to revert to the statically  defined cluster specs.

**Comment by casperakos** [comment]
With crossplane you will provision the eks cluster with a node group that will be responsible for running the Karpenter. Then karpenter will provision nodes based on your requirements but these nodes will not be managed by crossplane but karpenter.
